### DISCUSSION TITLE
Support for integrated Intel HD Graphics

### DISCUSSION THREAD

**Discussion by FelixGamerFG** [obsproject/obs-studio#11589] [Development Questions]
Even today, there are people with x64 architectures. And those people don't have a dedicated graphics card. Because their laptops don't have one... 

If you could make them give up the opportunity to use their Intel HD graphics for the encoder (which most of them are more powerful than their CPUs) that would be great. 

Please, small obs users need it.

**Comment by mihawk90** [comment]
Intel iGPUs and QuickSync have been supported since the dawn of time...?

**Reply by FelixGamerFG** [reply]
I know... but not all of us have the money or the conditions to buy another new PC. I understand that technology has to advance... but even so, small youtubers/streamers (mainly from Latin America) can't make videos/streaming with advanced versions of OBS Studios because we don't have the support of the integrated GPU. We also can't make videos with older versions of OBS because there are new tools in these decisions that we need.

**Reply by mihawk90** [reply]
I'm not sure I'm following you...

> Even today, there are people with x64 architectures.

Everything these days apart from a handful of ARM SOCs is x64.

> And those people don't have a dedicated graphics card.

This has nothing to do with that. A dedicated GPU doesn't depend on the bitness of the CPU.

> I know... but not all of us have the money or the conditions to buy another new PC.

Right... and what I'm saying is there is no need to when an iGPU with QuickSync exists.

> can't make videos/streaming with advanced versions of OBS Studios because we don't have the support of the integrated GPU.

Maybe I missed something, but I'm not aware that support for older models was dropped. Nvidia support for Kepler was dropped but I don't remember anything similar for Intel.

If OBS doesn't run on your PC/Laptop I'd suggest going through the support channels (which GitHub is not).

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Reply by FelixGamerFG** [reply]
Well, since version 28 it tells me that I need Nvidia or AMD. 

Searching all over the internet, the only solution that many people find is to change to OBS 27 or lower. 

And even so, there are many tools that we use such as: 

1. The internal audio capture source of software/games
2. the websoker (which I know exists before, but this one is better and more comfortable)
3. more compatible plugins

**Comment by RytoEX** [comment]
As far as I am aware, there should be no change in OBS Studio 28 that changes the availability of QSV. What did change in OBS Studio 28 was no longer supporting Windows 7, 8, and 8.1. If you are on Windows 7, 8, and 8.1, then OBS Studio 27.2.4 is still available for you to download and use. If you have an Intel CPU older than Broadwell, then OBS Studio 29.1.3 is the latest supported version of OBS Studio to use your Intel HD Graphics for encoding.

Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.
